proto/dsp/v1alpha2/control.proto

```diff
@@ -135,20 +141,26 @@ message ContractAcceptResponse {}
 message ContractAgreeRequest {
   string agreement = 1;
   string pid = 2;
+  bool fast_forward = 3;
+  bool want_notifications = 4;
 }
 
 message ContractAgreeResponse {}
 
 // ContractVerifyRequest contains the pid of the contract negotiation to verify.
 message ContractVerifyRequest {
   string pid = 1;
+  bool fast_forward = 2; // signals to RUN-DSP that it should fast forward the negotiation and not wait for any requests of the contract service.
```

Comment:
The term fast forward does not make sense to me, what would this actually mean.

And since the "want notifications" parameter would only be useful if fast forward is specified, wouldn't it be nicer to replace those two parameters with an enum?